acoustid/future/fpindex/updater.py

```
@@ -0,0 +1,172 @@
+    await conn.execute(
+        "SELECT * FROM pg_drop_replication_slot($1)",
+        SLOT_NAME,
+    )
+
+
+async def load_initial_data(conn: asyncpg.Connection) -> None:
+    """Load initial fingerprint data using a consistent snapshot"""
+
+    async with AsyncExitStack() as exit_stack:
+        await exit_stack.enter_async_context(
+            conn.transaction(isolation="repeatable_read", readonly=True)
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)